sdk/servicebus/service-bus/test/managementClient.spec.ts

```diff
@@ -1,112 +1,312 @@
 import chai from "chai";
 import chaiAsPromised from "chai-as-promised";
-import { delay, QueueClient, ServiceBusClient } from "../src";
+import Long from "long";
+import {
+  delay,
+  QueueClient,
+  ServiceBusClient,
+  TopicClient,
+  SubscriptionClient,
+  Receiver,
+  Sender,
+  ReceiveMode,
+  SessionReceiver
+} from "../src";
 import {
   TestClientType,
   getSenderReceiverClients,
   TestMessage,
-  getServiceBusClient
+  getServiceBusClient,
+  isSessionfulEntity
 } from "./utils/testUtils";
-chai.should();
+const should = chai.should();
 chai.use(chaiAsPromised);
 let sbClient: ServiceBusClient;
 
 async function afterEachTest(): Promise<void> {
   await sbClient.close();
 }
 
-describe("ManagementClient - disconnects", function(): void {
-  afterEach(async () => {
-    return afterEachTest();
-  });
+describe("ManagementClient", function(): void {
+  describe("ManagementClient - disconnects", function(): void {
```

Comment:
Am curious about the reason to add these tests in this PR.

Reply:
Wrapped both the describe blocks under a common `describe("ManagementClient")` block, Github is not able to recognize that and is showing the same code being added and deleted.

Reply:
Removed the `describe("ManagementClient")` for a cleaner diff.